pkg/registry/graph.go

```diff
@@ -0,0 +1,34 @@
+package registry
+
+type Package struct {
+	Name           string
+	DefaultChannel string
+	Channels       []Channel
+}
+
+type Channel struct {
+	Name            string
+	OperatorBundles []OperatorBundle
+	Head            BundleRef
+}
+
+type OperatorBundle struct {
+	BundlePath      string
+	Version         string // semver string
+	CsvName         string
+	ReplacesBundles []OperatorBundle
+	Replaces        []BundleRef
```

Comment:
```suggestion
	Replaces []*OperatorBundle
```
Why not this?

Reply:
After reviewing the rest, I think this could also be: 
```suggestion
	Replaces map[string]*OperatorBundle
```

or:

```suggestion
	Replaces map[BundleRef]*OperatorBundle
```